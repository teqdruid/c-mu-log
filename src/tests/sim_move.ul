/* checking the move with the simulator*/
size(15,15);
wall(3,4);
wall(5,4);
wall(3,8);
wall(12,4);
wall(3,9);
wall(3,12);
wall(13,13);
move("UP");
move("DOWN");
