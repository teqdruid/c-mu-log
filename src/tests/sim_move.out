x: Moving UP

==== Turn 1 ====
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
x . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 2 ====
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
x . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 3 ====
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
x . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 4 ====
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
x . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 5 ====
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
x . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 6 ====
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
x . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 7 ====
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
x . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 8 ====
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
x . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 9 ====
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
x . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 10 ====
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
x . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 11 ====
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
x . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 12 ====
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
x . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 13 ====
. . . . . . . . . . . . . . . 
x . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

==== Turn 14 ====
x . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . | . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . | . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
. . | . | . . . . . . | . . . 
. . . . . . . . . . . . . . . 
. . . . . . . . . . . . . . . 
# . . . . . . . . . . . . . . 

x: Moving UP

Simulation over: x hits the x margin and Game over!!! 

